app/src/routes/admin.php

```diff
@@ -201,6 +202,15 @@
             'method' => 'destroy',
             'middlewares' => [AdminMiddleware::class],
         ],
+
+        // Estadistiques
+        '/admin/grafics' => [
```

Comment:
Em pareix més raonable si son estadístiques, indica aquest diminutiu que no pas "gràfiques".

```suggestion
        '/admin/stats' => [
```